### PR TITLE
Added few more test stubs for discovery feature

### PR DESCRIPTION
**Pull request by sghai**
- Updated existing test stubs with correct test descriptions and removed typos at few places
- Added few more test stubs related to permissions and some invalid scenarios.

### CODE REVIEW

**Comment by elyezer** [comment]
ACK

File: tests/foreman/ui/test_discovery.py [hunk 1]
```diff
@@ -275,14 +333,90 @@ def test_update_discovery_rule_2(self):
...
 
-    def test_update_discovery_rule_3(self):
-        """@Test: Update the discovered host name with blank or long name
+    def test_update_discovery_prefix(self):
+        """@Test: Update the discovery_prefix parameter other than mac
 
         @Feature: Foreman Discovery
 
+        @Steps:
+        1. Goto settings -> Discovered tab -> discovery_prefix
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_discovery.py]: Nitpick: add a blank line between `@Steps:` and the list.

File: tests/foreman/ui/test_discovery.py [hunk 2]
```diff
@@ -275,14 +333,90 @@ def test_update_discovery_rule_2(self):
...
+
+        """
+
+    def test_add_new_discovery_fact(self):
+        """@Test: Add a new fact column to display on discovered host page
+
+        @Feature: Foreman Discovery
+
+        @Steps:
+        1. Goto settings -> Discovered tab -> discovery_fact_coloumn
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_discovery.py]: Nitpick: add a blank line between `@Steps:` and the list.

File: tests/foreman/ui/test_discovery.py [hunk 3]
```diff
@@ -275,14 +333,90 @@ def test_update_discovery_rule_2(self):
...
+        """
+
+    def test_add_invalid_discovery_fact(self):
+        """@Test: Add a new fact column with invalid fact to display on
+        discovered host page
+
+        @Feature: Foreman Discovery
+
+        @Steps:
+        1. Goto settings -> Discovered tab -> discovery_fact_coloumn
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_discovery.py]: Nitpick: add a blank line between `@Steps:` and the list.

File: tests/foreman/ui/test_discovery.py (hunk 1 above)
Comment (Ichimonji10) [inline tests/foreman/ui/test_discovery.py]: FYI, you can use unicode characters like the [rightwards arrow](http://www.fileformat.info/info/unicode/char/2192/index.htm) so long as the right comment is added to the first line of this file. You're not obliged to, of course, but they do look cool. :)

**Comment by Ichimonji10** [comment]
ACK pending comments

**Comment by sghai** [comment]
Comments are addressed.. thank you..

File: tests/foreman/ui/test_discovery.py (hunk 1 above)
Comment (elyezer) [inline tests/foreman/ui/test_discovery.py]: Good catch

**Pull request merged**